elections/templates/elections/election.html

```diff
@@ -0,0 +1,67 @@
+{% extends "home/base.html" %}
+{% load static %}
+
+{% block title %} Election {% endblock %}
+{% block section %}
+<section id="main">
+    {% if voter.has_voted %}
+        <h2>We have already received a ballot from {{voter}}.</h2>
+        <p>If you think this is a mistake, please contact us!</p>
+    {% else %}
+        <h3>Instructions</h3>
+        <p>
+            For each ballot, rank your choices by moving available candidates from
+            the left to a sorted position on the right.
```

Comment:
should we indicate that the top is the first choice?

Reply:
I guess more information isn't worse, but hopefully the little hint when you drag and drop will be intuitive enough too